language/German/strings.po

```diff
@@ -3634,73 +3695,73 @@ msgstr "[Keine]\n"
 #, c-format
 msgctxt "view-name"
 msgid "Screen %1$u Standard (%2$u:%3$u)"
-msgstr ""
+msgstr "Bildschirm %1$u Standard (%2$u:%3$u)"
 
 #: src/emu/render.cpp:1859
 #, c-format
 msgctxt "view-name"
 msgid "Screen %1$u Pixel Aspect (%2$u:%3$u)"
-msgstr ""
+msgstr "Bildschirm %1$u Pixel Aspect (%2$u:%3$u)"
 
 #: src/emu/render.cpp:1881 src/emu/render.cpp:2016
 msgctxt "view-name"
 msgid "Cocktail"
-msgstr ""
+msgstr "Cocktail"
 
 #: src/emu/render.cpp:2001
 msgctxt "view-name"
 msgid "Left-to-Right"
-msgstr ""
+msgstr "Von-Links-Nach-Rechts"
 
 #: src/emu/render.cpp:2002
 msgctxt "view-name"
 msgid "Left-to-Right (Gapless)"
-msgstr ""
+msgstr "Von-Links-Nach-Rechts (lückenlos)"
```

Comment:
I don't think the dashes are correct in this context. "Von links nach rechts" would be better I think. Same for the next few items

Reply:
Fair enough, removed